src/components/RightSidebar/RightSidebar.vue

```diff
@@ -214,6 +215,10 @@ export default {
 				this.conversationName = this.conversation.displayName
 			}
 		},
+
+		token() {
+			this.$refs.participantsTab.$el.scrollTop = 0
```

Comment:
could also be in watch conversation?
One watcher less?

Reply:
good point. hmmm, I feel like watching a string is more safe than an object changing, but it might be just me

Reply:
adjusted

Reply:
seems watching the object is causing side effects now, my intuition was right: https://github.com/nextcloud/spreed/issues/5904